reaktive/src/nativeCommonTest/kotlin/com/badoo/reaktive/looperthread/LooperThreadTest.kt

```diff
@@ -29,27 +29,12 @@ class LooperThreadTest {
         }
 
         lock.synchronized {
-            condition.waitForOrFail(5000L, isExecuted::value)
+            condition.waitForOrFail(5_000_000_000L, isExecuted::value) // Wait for 5 secs
```

Comment:
I hope we will fix it after copy-pasting TimeUnits from klock.

Reply:
Or by using coming Duration from stdlib